lightkurve/targetpixelfile.py

```diff
@@ -489,9 +489,55 @@ def centroids(self, **kwargs):
                       LightkurveWarning)
         return self.estimate_centroids(**kwargs)
 
-    def estimate_centroids(self, aperture_mask='pipeline'):
+    def estimate_centroids(self, aperture_mask='pipeline', method='moments'):
```

Comment:
Let's make `quadratic` the default `method`, or do you think `moments` is significantly more robust?